packages/ffe-cards/less/cards.less

```diff
@@ -1,3 +1,4 @@
+@import 'common-card-styling';
 @import 'card-base';
```

Comment:
Hva er grunnen til at vi forsatt må ha card-base?  Sånn utenom display: block og padding? Er det forskjellige bruksområder?

Reply:
Jeg tok en titt på i våre interne repos og der blir CardBase mye brukt. Skulle gjerne fjerne CardBase, men det skaper masse jobb for andre.